src/api/app/models/user.rb

```diff
@@ -77,7 +77,7 @@ class User < ApplicationRecord
   scope :with_login_prefix, ->(prefix) { where('login LIKE ?', "#{prefix}%") }
   scope :active, -> { confirmed.or(User.unscoped.where(state: :subaccount, owner: User.unscoped.confirmed)) }
   scope :staff, -> { joins(:roles).where('roles.title = ?', 'Staff') }
-  scope :not_staff, -> { where.not(id: User.staff.pluck(:id)) }
+  scope :not_staff, -> { where.not(id: User.unscoped.staff.pluck(:id)) }
```

Comment:
Shouldn't this be `default_scoped`?

Reply:
But this model does not have a default scope, so I think no

Reply:
There isn't any `default_scope`defined in User model, so `unscoped` is correct here.

Reply:
Okay, I understood the `To instead access the full set of models, as Rails 6.1 will, use `BsRequest.unscoped`, 
or `BsRequest.default_scoped` if a model has default scopes.` as `either use unscoped or default_scoped`